eve/main.yml

```diff
@@ -34,6 +35,11 @@ stages:
         method: clobber
         retryFetch: true
         haltOnFailure: true
+    - ShellCommand:
+        name: 'Check syntax'
+        haltOnFailure: true
+        command: |-
+          tox -e pep8
```

Comment:
Could also be a Docker container with all deps pre-installed, even using Alpine.

Reply:
Yes was doing that ;)